lib/safe-pg-migrations/plugins/idempotent_statements.rb

```diff
@@ -23,19 +20,24 @@ module IdempotentStatements
     end
 
     ruby2_keywords def add_column(table_name, column_name, type, *)
-      return super unless column_exists?(table_name, column_name)
+      if column_exists?(table_name, column_name) && !column_exists?(table_name, column_name, type)
```

Comment:
does this also check for properties like NOT NULL or DEFAULT ?

Reply:
as it is not but I think that the method `column_exists` can do that
DO you want me to add those case also? 

If yes I think I will merge this PR and open another one if that's ok for you?

Reply:
created a ticket to handle this later as discussed